### PR TITLE
added alert and button

### PR DESCRIPTION
**Pull request by Swapnilkota108**

### CODE REVIEW

File: components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+.alert{
+    height: 50px;
+    width: 90%;
+    display: flex;
+    align-items: center;
+    background-color: #fff;
+    color: #fff;
```
Comment (srishti-maurya) [inline components/Alert/alert.css]: suggestion: you can use variable names here

**Review by srishti-maurya — APPROVED** [review]
Suggestion: You can add a preview link for ease.

**Review by nshanbhag20001412 — APPROVED** [review]
looks good

File: components/Alert/alert.css (hunk 1 above)
Comment (Swapnilkota108) [reply components/Alert/alert.css]: Thanks for the suggestion.

**Comment by Swapnilkota108** [comment]
> looks good

**Pull request merged**